lib/static/css/main.css

```diff
@@ -1545,8 +1545,28 @@ input::placeholder {
 }
 
 @media only screen and (max-width: 735px) {
-  .docs-prevnext {
```

Comment:
Deleting this means the class here is useless
https://github.com/notjaril/Docusaurus/blob/f9151ca17ac424d0102d65faedef78db15ef4cbd/lib/core/DocsLayout.js#L90

```js
<div className="docs-prevnext">
```

Reply:
The docs-prevnext class is still styled with: 

```css
/* Start of Docs Navigation */
.docs-prevnext {
  margin: 20px 0;
}
```

The intent with removing the doc-prevnext styling within the media query was so the buttons' top and bottom margins were displayed properly, where it wasn't being display properly with a fixed doc-prevnext height. 

![docs-prevnext](https://user-images.githubusercontent.com/15959269/44725319-00866900-aaa3-11e8-9eb3-5d5e92bb906d.png)

Does this make sense? Let me know if I missed something.

Reply:
This is correct. I missed that :+1: